bundler.go

```diff
@@ -538,12 +554,17 @@ func (b *Bundler) provisionVendorAstilectron() (err error) {
 			return b.ctx.Err()
 		}
 	}
-	return b.provisionVendorZip(astilectron.AstilectronDownloadSrc(), p, filepath.Join(b.pathVendor, zipNameAstilectron))
+	return b.provisionVendorZip(astilectron.AstilectronDownloadSrc(b.versionAstilectron), p, filepath.Join(b.pathVendor, zipNameAstilectron))
 }
 
 // provisionVendorElectron provisions the electron vendor zip file
-func (b *Bundler) provisionVendorElectron(oS, arch string) error {
-	return b.provisionVendorZip(astilectron.ElectronDownloadSrc(oS, arch), filepath.Join(b.pathCache, fmt.Sprintf("electron-%s-%s-%s.zip", oS, arch, astilectron.VersionElectron)), filepath.Join(b.pathVendor, zipNameElectron))
+func (b *Bundler) provisionVendorElectron(oS, arch, versionElectron string) error {
+	astilog.Debugf("provisionVendorElectron versionElectron %s", versionElectron)
+
+	return b.provisionVendorZip(
+		astilectron.ElectronDownloadSrc(oS, arch, b.versionElectron),
+		filepath.Join(b.pathCache, fmt.Sprintf("electron-%s-%s-%s.zip", oS, arch, versionElectron)),
```

Comment:
Use `b.versionElectron` instead of `versionElectron`

Reply:
Set to use versions in Bundler